PetAdoption-iOS/PetAdoption-iOS.xcodeproj/project.pbxproj

```diff
@@ -47,6 +47,33 @@
 		D0950B131C61A667002E198C /* PetImageCollectionViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = D0950B121C61A667002E198C /* PetImageCollectionViewCell.swift */; };
 		D0E536791C70405D004794D8 /* SearchViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = D0E536781C70405D004794D8 /* SearchViewController.swift */; };
 		D0E5367B1C70412A004794D8 /* PetListingDetailViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = D0E5367A1C70412A004794D8 /* PetListingDetailViewController.swift */; };
+		D4F2A7321D6A081F00E92647 /* PetAdoptionTransportKit.h in Headers */ = {isa = PBXBuildFile; fileRef = D4F2A7311D6A081F00E92647 /* PetAdoptionTransportKit.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		D4F2A7391D6A081F00E92647 /* PetAdoptionTransportKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D4F2A72F1D6A081F00E92647 /* PetAdoptionTransportKit.framework */; };
+		D4F2A7401D6A081F00E92647 /* PetAdoptionTransportKitTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4F2A73F1D6A081F00E92647 /* PetAdoptionTransportKitTests.swift */; };
+		D4F2A7441D6A081F00E92647 /* PetAdoptionTransportKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D4F2A72F1D6A081F00E92647 /* PetAdoptionTransportKit.framework */; };
+		D4F2A7451D6A081F00E92647 /* PetAdoptionTransportKit.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = D4F2A72F1D6A081F00E92647 /* PetAdoptionTransportKit.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		D4F2A7511D6A0D6800E92647 /* Alamofire.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D4F2A7501D6A0D6800E92647 /* Alamofire.framework */; };
```

Comment:
What I mean is, should Alamofire get removed from the Podfile?  Will leaving it there cause any conflicts down the road?